packages/core/src/components/ui/VirtualTable/_VirtualTable.scss

```diff
@@ -0,0 +1,58 @@
+@import "../../../common/styles/colors";
+
+.ReactVirtualized__Table__headerRow {
+  border-bottom: none;
+  padding-right: 0 !important;
+  height: 48px;
+}
+
+.ReactVirtualized__Table__headerColumn {
+  font-weight: normal;
+
+  > * {
+    font-size: 11px !important;
+  }
+}
+
+.ReactVirtualized__Table__headerTruncatedText {
+  margin-right: 4px;
+  font-weight: bold;
+  font-size: 13px;
+  text-transform: none;
+}
+
+.ReactVirtualized__Table__row {
+  color: $white;
+
+  > * {
+    font-size: 13px;
+    line-height: 20px !important;
+  }
+
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.vt-container {
+  display: flex;
+
+  .hfui-table {
+    flex: 1 1 auto;
+    height: calc(100vh - 35px);
+    width: 100vw;
```

Comment:
please change vw to %, since all comps in ufx are responsive relative to its parent width and not the screen width. (if we need vw for hfx ui, customise it in hf ui scss)